src/game/TileEngine/Isometric_Utils.h

```diff
@@ -82,8 +82,9 @@ INT16 CardinalSpacesAway(INT16 sOrigin, INT16 sDest);
 bool FindHigherLevel(SOLDIERTYPE const*, UINT8* out_direction = 0);
 bool FindLowerLevel(SOLDIERTYPE const*, UINT8* out_direction = 0);
 
-INT16 QuickestDirection(INT16 origin, INT16 dest);
-INT16 ExtQuickestDirection(INT16 origin, INT16 dest);
+template<UINT8 maxDistance = 4>
+INT8 QuickestDirection(UINT8 origin, UINT8 dest);
+INT8 ExtQuickestDirection(UINT8 origin, UINT8 dest);
```

Comment:
ExtQuickestDirection has only one user, so I suggest we add a defaulted maxDistance parameter to QuickestDirection instead of templating it and remove ExtQuickestDirection altogether.